cdc/redo/reader/reader.go

```diff
@@ -363,10 +364,22 @@ func (h logHeap) Less(i, j int) bool {
 		return false
 	}
 
-	if h[i].data.RedoRow.Row.CommitTs == h[j].data.RedoRow.Row.CommitTs &&
-		h[i].data.RedoRow.Row.StartTs < h[j].data.RedoRow.Row.StartTs {
-		return true
+	if h[i].data.RedoRow.Row.CommitTs == h[j].data.RedoRow.Row.CommitTs {
+		if h[i].data.RedoRow.Row.StartTs < h[j].data.RedoRow.Row.StartTs {
```

Comment:
remove useless else branch:
```suggestion
		if h[i].data.RedoRow.Row.StartTs != h[j].data.RedoRow.Row.StartTs {
		return h[i].data.RedoRow.Row.StartTs < h[j].data.RedoRow.Row.StartTs
		}
		...
```

Reply:
Good Point!